### PR TITLE
add embed for txns and scripts

### PR DESCRIPTION
**Pull request by Deewai**
This PR makes it easy for other repos to import transactions and scripts from this repo so we can avoid copying over txns and scripts.
Note: no contract was modified in this PR. All contracts in this PR was copied over.

### CODE REVIEW

File: contracts/PackNFT.cdc [hunk 1]
```diff
@@ -0,0 +1,496 @@
+import Crypto
```
Comment (Deewai) [inline contracts/PackNFT.cdc:1]: this is copied over from studio-platform-smart-contracts

File: contracts/imports/IPackNFT.cdc [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import Crypto
```
Comment (Deewai) [inline contracts/imports/IPackNFT.cdc:1]: this is copied over from studio-platform-smart-contracts

File: embed.go [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+		//DapperUtilityCoinContractAddress: dapperUtilityCoinAddress,
+		MetadataViewsAddress:    metadataViewsAddress,
+		ViewResolverAddress:     viewResolverAddress,
+		BurnerAddress:           burnerAddress,
+		IPackNFTContractAddress: iPackNftContractAddress,
+		PackNFTAddress:          packNftContractAddress,
+	}, nil
+}
+
+func contractAddressHex(flowConf *config.Config, name, network string) (string, error) {
```
Comment (judezhu) [inline embed.go:199]: We don't need these `script` or `transaction` processor. These script/transaction processors are overly complicated. Can you sync tomorrow

Comment (Deewai) [reply embed.go:199]: Yes, let's sync tomorrow. I also thought it was way too complicated

**Review by judezhu — APPROVED** [review]

**Pull request merged**